app/services/to_cocina/work/contributors_mapper.rb

```diff
@@ -19,27 +19,37 @@ def call
           # First is always status: "primary"
           primary = index.zero?
           if contributor.person?(with_names: true)
-            CocinaGenerators::Description.person_contributor(
-              forename: contributor.first_name,
-              surname: contributor.last_name,
-              role: contributor.person_role,
-              primary:,
-              orcid: contributor.orcid
-            )
+            person_for(contributor:, primary:)
```

Comment:
is `person_for` expected here whereas organization is `organization_form`?

Reply:
Typo. Fixed.